### ISSUE TITLE
Chrome Extension Inserts Image Filename for alt text when no alt text is Provided

### ISSUE THREAD

**Issue by macmanx2** [MLTSHP/mltshp_plugins#11]
Using https://macmanx.com/2016/10/03/automattic-meetup-2016/ as an example (I know, shame on me, but it came in handy today.), if you right-click an alt-less image and choose "Save this image on MLTSHP" with the Chrome extension, the image's filename is inserted as alt text, which is quite unhelpful as alt text. It would be better if the alt text were simply blank or said "alt text not provided" if something needs to be in the field for the extension.

This is not a problem in the Firefox add-on, as the alt field is left blank when the image has no alt text.

<img width="858" alt="Screenshot 2024-08-22 at 1 23 44 PM" src="https://github.com/user-attachments/assets/b9c07c05-f6d7-4058-a65c-f83fa57ff914">

**Comment by dphiffer** [comment]
Inspecting the example page, I see that the image in question does have an `alt` attribute. Including the HTML here for reference:
```
<img decoding="async" class="" data-attachment-id="16563" data-orig-file="https://macmanx.com/wp-content/uploads/2016/10/a8cgmwhistler-1.jpg" data-orig-size="4032,3024" data-comments-opened="1" data-image-meta="{&quot;aperture&quot;:&quot;2.2&quot;,&quot;credit&quot;:&quot;&quot;,&quot;camera&quot;:&quot;iPhone 6s&quot;,&quot;caption&quot;:&quot;&quot;,&quot;created_timestamp&quot;:&quot;1473929134&quot;,&quot;copyright&quot;:&quot;&quot;,&quot;focal_length&quot;:&quot;4.15&quot;,&quot;iso&quot;:&quot;25&quot;,&quot;shutter_speed&quot;:&quot;0.0008058017727639&quot;,&quot;title&quot;:&quot;&quot;,&quot;orientation&quot;:&quot;0&quot;}" data-image-title="a8cgmwhistler-1" data-image-description="" data-medium-file="https://i0.wp.com/macmanx.com/wp-content/uploads/2016/10/a8cgmwhistler-1.jpg?fit=300%2C225&amp;quality=89&amp;ssl=1" data-large-file="https://i0.wp.com/macmanx.com/wp-content/uploads/2016/10/a8cgmwhistler-1.jpg?fit=1024%2C768&amp;quality=89&amp;ssl=1" src="https://i0.wp.com/macmanx.com/wp-content/uploads/2016/10/a8cgmwhistler-1.jpg?w=330&amp;h=247&amp;quality=89&amp;ssl=1" srcset="https://i0.wp.com/macmanx.com/wp-content/uploads/2016/10/a8cgmwhistler-1.jpg?w=4032&amp;quality=89&amp;ssl=1 4032w, https://i0.wp.com/macmanx.com/wp-content/uploads/2016/10/a8cgmwhistler-1.jpg?resize=300%2C225&amp;quality=89&amp;ssl=1 300w, https://i0.wp.com/macmanx.com/wp-content/uploads/2016/10/a8cgmwhistler-1.jpg?resize=768%2C576&amp;quality=89&amp;ssl=1 768w, https://i0.wp.com/macmanx.com/wp-content/uploads/2016/10/a8cgmwhistler-1.jpg?resize=1024%2C768&amp;quality=89&amp;ssl=1 1024w, https://i0.wp.com/macmanx.com/wp-content/uploads/2016/10/a8cgmwhistler-1.jpg?resize=770%2C578&amp;quality=89&amp;ssl=1 770w, https://i0.wp.com/macmanx.com/wp-content/uploads/2016/10/a8cgmwhistler-1.jpg?w=2000&amp;quality=89&amp;ssl=1 2000w, https://i0.wp.com/macmanx.com/wp-content/uploads/2016/10/a8cgmwhistler-1.jpg?w=3000&amp;quality=89&amp;ssl=1 3000w,https://i0.wp.com/macmanx.com/wp-content/uploads/2016/10/a8cgmwhistler-1.jpg?w=330&amp;h=247&amp;quality=89&amp;ssl=1&amp;resize=660%2C494&amp;_jb=custom 2880w" width="330" height="247" loading="lazy" data-original-width="330" data-original-height="247" itemprop="http://schema.org/image" title="a8cgmwhistler-1" alt="a8cgmwhistler-1" style="width: 330px; height: 247px;" sizes="auto">
```

**Comment by macmanx2** [comment]
Thanks @dphiffer I was so sure I checked that, but I was also balancing a lot of plates at the time, it must have just slipped my mind.

Closing this as WFM.
